platform.go

```diff
@@ -40,6 +40,27 @@ const (
 	// See the Service Binding Specification for Kubernetes for more details - https://k8s-service-bindings.github.io/spec/
 	EnvServiceBindings = "SERVICE_BINDING_ROOT"
 
+	// EnvBuildpackDirectory is the name of the environment variable that contains the path to the buildpack
```

Comment:
These other constants should probably be in different files?

Reply:
I honestly wasn't sure what makes sense (kind of paralyzed by choice, lol).

To me build.go/detect.go are where they are used most, but perhaps you could make a case they go in buildpack.go since they are part of the buildpack spec, or possibly we should have a const.go (or something like that) where all the consts in `libcnb` package go (I think I've seen that before in go projects).

In the end, I put them here to be colocated with service binding root since I followed the pattern `Env*` for the name of the const. All those locations end up in `libcnb` package, so it didn't seem to make a huge difference from the consumer side.

Do you have a preference where? I don't, so if you want me to move them, I'd be happy to.

Reply:
I am going to approve and merge this. We can move the variables around in a different PR.